WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsPresentedViewViewModel.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+protocol CardReaderSettingsPresentedViewModel {
+    func shouldShow() -> Bool
+}
+
+/// A tuple containing a reference to the view model for a presented view and the related view controller's class name.
```

Comment:
Would you consider declaring this a a struct? Structs are super cheap, and you could make it implement Equatable/Comparable which might help clean code that looks up into the array of these later.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4066/commits/d333edc07151bed8e136534cd197070871ceff57